### PR TITLE
Fixed #4839 - Reversions in the include language file

### LINKED ISSUE #4839: Reversions in the include language file?
#### Issue

There are too many differences in the /include/language/en_us.lang.php between the latest branch (develop / staging/7.10 and hotfix)
Note: those differences are not related to formatting clean up (All files compared has the formating revised) or new strings as the ones for Survey module.
This is related to strings that was removed in latest versions that are added back.

Develop and staging are closer to each other but the different strings are a reversion of previous deleted strings. I think this happen after some merge was done from a file not updated after hotfix but from an old master branch (but with code format revised), so many strings was added back.

Develop: https://github.com/salesagility/SuiteCRM/blob/develop/include/language/en_us.lang.php
Staging: https://github.com/salesagility/SuiteCRM/blob/staging/7.10/include/language/en_us.lang.php

I consider the Hotfix as the cleaner file, where some old string was removed (and its the reference for Crowdin)
Hotfix: https://github.com/salesagility/SuiteCRM/blob/hotfix/include/language/en_us.lang.php

Now the question @Dillon-Brown is the reversion on developer branch an intended reversion or just an error?

#### Possible Fix

Revise the developer branch

#### Context

Allow 7.10 translations ready to deploy

### PR DESCRIPTION
**Pull request by Dillon-Brown**
## Description

Fixes a diff issue between hotfix and develop. This PR adds in missing strings and removes duplicates + unused.

## Motivation and Context

Issue reference: #4839

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: include/language/en_us.lang.php [hunk 1]
```diff
@@ -1688,19 +1494,15 @@
...
-    'LBL_LISTVIEW_ALL' => 'All',
     'LBL_LISTVIEW_OPTION_SELECTED' => 'Selected Records',
     'LBL_LISTVIEW_SELECTED_OBJECTS' => 'Selected: ',
 
     'LBL_LOCALE_NAME_EXAMPLE_FIRST' => 'David',
     'LBL_LOCALE_NAME_EXAMPLE_LAST' => 'Livingstone',
     'LBL_LOCALE_NAME_EXAMPLE_SALUTATION' => 'Dr.',
     'LBL_LOCALE_NAME_EXAMPLE_TITLE' => 'Code Monkey Extraordinaire',
-    'LBL_LOGIN_TO_ACCESS' => 'Please sign in to access this area.',
     'LBL_LOGOUT' => 'Log Out',
```
Comment (horus68) [inline include/language/en_us.lang.php]: LBL_LOGOUT  = duplicated string (keep the one in line 1058)

File: include/language/en_us.lang.php [hunk 2]
```diff
@@ -1197,41 +1121,28 @@
...
     'LBL_EMAIL_DETAILS' => 'Details',
-    'LBL_EMAIL_DISPLAY_MSG' => 'Displaying email(s) {0} - {1} of {2}',
-    'LBL_EMAIL_ADDR_DISPLAY_MSG' => 'Displaying email address(es) {0} - {1} of {2}',
 
-    'LBL_EMAIL_EDIT_CONTACT' => 'Edit Contact',
     'LBL_EMAIL_EDIT_CONTACT_WARN' => 'Only the Primary address will be used when working with Contacts.',
-    'LBL_EMAIL_EDIT_MAILING_LIST' => 'Edit Mailing List',
 
     'LBL_EMAIL_EMPTYING_TRASH' => 'Emptying Trash',
     'LBL_EMAIL_DELETING_OUTBOUND' => 'Deleteting outbound server',
```
Comment (horus68) [inline include/language/en_us.lang.php]: see PR https://github.com/salesagility/SuiteCRM/pull/4837 for english typos in 
LBL_EMAIL_DELETING_OUTBOUND
LBL_EMAIL_CLEARING_CACHE_FILES

**Comment by horus68** [comment]
Any decision on this? This is a major file and there shouldn't be so many differences on a reference file as there are others creating PR's using one or other version of the file.

**Comment by Dillon-Brown** [comment]
@horus68 This PR will need properly code reviewed and assessed before it can be merged but I don't think there will be any issues. I just checked the diff between hotfix and develop and corrected any strings that were removed from hotfix but not develop so if there were to be any missing labels as a result of this PR we would be currently seeing them in hotfix.

**Comment by horus68** [comment]
@Dillon-Brown file has conflicts!

**Pull request merged**